### PR TITLE
refactor: pass lighthouse accessibility audits

### LINKED ISSUE #259: Improving Google Lighthouse SEO score
https://github.com/gokarna-theme/gokarna-hugo/issues/250 talks about improving performance based on Google lighthouse score. Reviewing the [lightscore score](https://pagespeed.web.dev/analysis/https-gokarna-hugo-netlify-app/w90db0qrkg?form_factor=desktop), there's a scope of improving SEO to improve our visibility as a theme. 

image

image

cc @jamesericdavidson

### PR DESCRIPTION
**Pull request by jamesericdavidson**
- [x] sequential headings
  - [x] css fontSize changes for h3 => h2 change? @yashmehrotra @526avijitgupta 
- [x] give names to links (screen readers)

### CODE REVIEW

**Comment by jamesericdavidson** [comment]
@526avijitgupta Passes locally 

image

**Comment by jamesericdavidson** [comment]
@526avijitgupta Did the Netlify deploy bot break when migrating the repo to an organisation?

**Comment by 526avijitgupta** [comment]
@jamesericdavidson oh I just commented regarding the same thing here :D :https://github.com/gokarna-theme/gokarna-hugo/pull/261#issuecomment-2439663575

Let's verify in one of your open PRs if the deploy previews are working again 🤞

File: layouts/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
     <h1>{{ .Site.Title }}</h1>
     {{ if isset .Site.Params "description" }}
-        <h3>{{ .Site.Params.Description }}</h3>
+        <h2>{{ .Site.Params.Description }}</h2>
```
Comment (526avijitgupta) [inline layouts/index.html:11]: This increases the font size of the about description. I tried and setting `font-size: 1.2em;` makes it back to normal. Can you please add the css rule?
```css
.home-about h2 {
  font-size: 1.2em;
}
```

**Review by 526avijitgupta — CHANGES_REQUESTED** [review]

File: layouts/index.html (hunk 1 above)
Comment (jamesericdavidson) [reply layouts/index.html:11]: Before: 

image

After: 

image

**Comment by jamesericdavidson** [comment]
@526avijitgupta I think review is failing because it's pointed at `index.html`

Deploy preview is passing:

image

**Review by 526avijitgupta — APPROVED** [review]
Thanks @jamesericdavidson 🚀

**Pull request merged**